test/test_train_imagenet.py

```diff
@@ -164,10 +166,13 @@ def test_loop_fn(model, loader, device, context):
     return correct / total_samples
 
   accuracy = 0.0
+  writer = SummaryWriter(log_dir='/tmp/imagenet_tensorboard')
```

Comment:
Could we add FLAGS.logdir? What do you think?

Reply:
Defaulted to FLAGS.logdir